account_budget_template/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Account Budget Template",
+    "version": "11.0.1.2.0",
+    "category": "Accounting",
+    "license": "AGPL-3",
+    "author": "AvanzOSC,"
+              "Odoo Community Association (OCA)",
+    "website": "http://www.avanzosc.es",
```

Comment:
URL should be 'https://github.com/OCA/account-financial-tools'

Reply:
Done!